### ISSUE TITLE
Functional Tests for Streaming Extensions Library

### ISSUE THREAD

**Issue by DDEfromOR** [microsoft/botbuilder-js#1188]
There are currently no end to end functional tests for the streaming extensions library integration with Bot Builder. This means if changes to Bot Builder break compatibility with streaming extensions we won't know until it's too late. 

Specific test cases that came to mind while writing this:

- Round trip message using streaming connections. (Named Pipe w/ DL ASE, WebSocket w/ modified DL ASE or DLS)
- Round trip message using HTTP through the streaming adapter. (New adapter with classic workflow.)
- Round trip message with attachments using streaming connection. (Named Pipe w/ DL ASE, WebSocket w/ modified DL ASE or DLS)

**Comment by ckkashyap** [comment]
PR created - https://github.com/microsoft/botbuilder-js/issues/1188

**Comment by cleemullins** [comment]
#1188 is an issue, not a PR. Is there a PR that matches?

**Comment by cleemullins** [comment]
@DDEfromOR @ckkashyap Can I get a status update for this?

**Comment by DDEfromOR** [comment]
Bumping this to after R7. It requires changes on the service side and then having a partner team pull those changes in to their product, which isn't likely to land before R7 due to blackouts.

**Comment by gabog** [comment]
Hi @DDEfromOR, what should we do about this one? Close it? Address it in R10?

**Comment by DDEfromOR** [comment]
Hi Gabo, the work to get the necessary support into the CLI to make these tests possible is finally nearing completion. We could either target R10 or close it and open a new issue once the CLI work is done. (I'm not clear on the rules here, I originally opened this just to make sure the functional tests didn't get forgotten.)

**Comment by mrivera-ms** [comment]
@mrivera-ms create a parent issue in the SDK repo.

**Comment by Zerryth** [comment]
[End-to-End Streaming tests in botbuiler-js](https://github.com/microsoft/botbuilder-js/pull/2569) partially addresses this issue, specifically Daniel's 1st bullet:

> Round trip message using streaming connections. (Named Pipe w/ DL ASE, WebSocket w/ modified DL ASE or DLS)

**Comment by cleemullins** [comment]
This issue is 11 months old. Removing the P0 and R10 labels, and moving to backlog.

**Comment by Zerryth** [comment]
Per Daniel, the E2E streaming tests implemented in R10 cover all of the concerns he had: using named pipes, websockets, and backwards compatibility with the production channel.

Closing as [resolved ](https://github.com/microsoft/botbuilder-js/pull/2569) now
